opam

```diff
@@ -7,7 +7,7 @@ bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
 tags:         "org:mirage"
 license:      "ISC"
 
-build:   [["./configure"] [make]]
+build:   [["env" "CONDUIT_DISCOVER_VERBOSE=1" "./configure"] [make]]
```

Comment:
I don't mind if we get rid of this variable and always write the "verbose" output as well.